pkg/build/controller/controller.go

```diff
@@ -182,7 +182,7 @@ func (bc *BuildController) nextBuildPhase(build *buildapi.Build) error {
 func (bc *BuildController) resolveOutputDockerImageReference(build *buildapi.Build) (string, error) {
 	outputTo := build.Spec.Output.To
 	if outputTo == nil || outputTo.Name == "" {
-		return "", nil
+		return "no_repo/no_output_default_tag", nil
```

Comment:
sorry, @bparees  is right, i wasn't paying attention to `if len(s.build.Status.OutputDockerImageReference) > 0` which is voided by this change, i will revert the change.